EIPS/eip-draft_title_abbrev.md

```diff
@@ -0,0 +1,74 @@
+---
+eip: <to be assigned>
+title: EthPM URI Specification
+author: Nick Gheorghita (@njgheorghita), Piper Merriam (@pipermerriam), g. nicholas d'andrea (@gnidan), Benjamin Hauser (@iamdefinitelyahuman)
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-09-04
+requires: EIP 2678
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+A custom [URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) scheme to identify an [EthPM](https://docs.ethpm.com/) registry, package, release, or specific contract asset within a release.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+When interacting with the EthPM ecosystem, users and tooling can benefit from a URI scheme to identify EthPM assets. Being able to specify a package, registry, or release with a single string makes simplifies the steps required to install, publish, or distribute EthPM packages.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
+`scheme://registry_address:chain_id/package_name@package_version/json_pointer`
+
+#### `scheme`
+- Required
+- Must be one of `ethpm` or `erc2678`. If future versions of the EthPM packaging standard are designed and published via the ERC process, those ERCs will also be valid schemes.
+
+#### `registry_address`
+- Required
+- This **should** be either an ENS name or a 0x-prefixed, checksummed address. ENS names are more suitable for cases where mutability of the underlying asset is acceptable and there is implicit trust in the owner of the name. 0x prefixed addresses are more preferable in higher security cases to avoid needing to trust the controller of the name.
+
+#### `chain_id`
+- Optional
+- Integer representing the chain id on which the registry is located
+- If omitted, defaults to `1` (mainnet).
+
+#### `package_name`
+- Optional
+- String of the target package name
+
+#### `package_version`
+- Optional
+- String of the target package version
+- If the package version contains any [url unsafe characters](https://www.werockyourweb.com/url-escape-characters/), they **must** be safely escaped
```

Comment:
If possible, inline the allowed/unsafe characters directly rather than linking to some random external place on the internet that is likely to go away at some point.  We want specifications to be self-standing as much as possible.  Another option is to include external dependencies as an asset in `../assets/eip-2942/whatever.ext`

Reply:
I'm not sure this use-case warrants it's own definition of a fairly universal standard. I've updated the link to wikipedia which should be more resilient. If you feel strongly otherwise, let me know and I'll inline the list.

Reply:
I would still recommend inlindng over a Wikipedia link.  I also wouldn't be terribly opposed to just referring to unsafe URL characters by name and letting people Google for it.  This isn't a blocker for draft merge either way.